buildstockbatch/postprocessing.py

```diff
@@ -555,12 +555,12 @@ def combine_results(fs, results_dir, cfg, do_timeseries=True):
                 ts_out_loc = f"{ts_dir}/upgrade={upgrade_id}/"
             else:
                 assert isinstance(fs, S3FileSystem)
-                ts_out_loc = f"s3://{ts_dir}/upgrade={upgrade_id}/"
+                ts_out_loc = f"s3://{ts_dir}/upgrade={upgrade_id}"
```

Comment:
I believe that this can be simplified...

1. This trailing slash shouldn't be there for the LocalFileSystem either. What uses this is `concat_and_normalize`, which appends a "/" before also appending the output filename (which is good design). So, as-is, there will be a double-slash in the path for LocalFileSystem (though most local filesystems treat them like a single slash and work just fine). Like with S3FileSystem, a trailing slash _doesn't_ work for GCSFileSystem.

2. The scheme ("s3://") is likely unnecessary since anywhere the path is used, it's used with `fs` (which will already be S3FileSystem). The scheme is not necessary for GCSFilesystem.

So, if you remove the extraneous slash from the LocalFileSystem and the scheme from the S3FileSystem, `ts_out_loc` will always be just `f"{ts_dir}/upgrade={upgrade_id}"`; i.e., you can get rid of the if/else and it'll work for all three filesystems (local, S3, and GCS).

Reply:
Good point. I made the change.